packages/shared/src/components/fields/MarkdownInput/index.tsx

```diff
@@ -278,7 +280,7 @@ function MarkdownInput(
             wrapper={(children) => (
               <span
                 className={classNames(
-                  'grid grid-cols-3 gap-3',
+                  'flex gap-3',
```

Comment:
Just a question, why did we end up moving to `flex gap`, is the `grid grid-cols-n` doesn't work as how it used to anymore?

Reply:
I think this was a fault to be honest.
Don't know why that was chosen, but on one screen we only render 1 item, which made it align super weird (in the first of 3 columns)

Flex does exactly this, but automatic 😅
Love to hear if it has a particular use-case I'm unaware off

Reply:
Nothing really. Was just curious about the change if it was tailwind update related. Flex does the job better anyways, and won't limit us to the number of elements. Way to go! 🚀